templates/taxi/car_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base_generic.html" %}
+{% load crispy_forms_tags %}
+
+{% block content %}
+  <h2>Create or Update Car</h2>
+  <form method="post">
+    {% csrf_token %}
+    {{ form|crispy }}
+    <button type="submit" class="btn btn-primary">Submit</button>
```

Comment:
As per the checklist, it is required to add a `Cancel` button apart from the `Submit` button. The `Cancel` button will lead to the previous page the user was on. Please add it.